Content.Server/Bible/BibleSystem.cs

```diff
@@ -194,6 +197,17 @@ private void OnFamiliarDeath(EntityUid uid, FamiliarComponent component, MobStat
             }
         }
 
+        /// <summary>
+        /// When the familiar spawns, set its source to the bible.
+        /// </summary>
+        private void OnSpawned(EntityUid uid, FamiliarComponent component, GhostRoleSpawnerUsedEvent args)
+        {
+            if (!TryComp<SummonableComponent>(Transform(args.Spawner).ParentUid, out var summonable))
+                return;
+
+            component.Source = (EntityUid) summonable.Owner;
```

Comment:
```suggestion
            component.Source = summonable.Owner;
```
dont think it needs a cast?